modules/core/include/opencv2/core/cuda/common.hpp

```diff
@@ -95,26 +97,6 @@ namespace cv { namespace cuda
         {
             return (total + grain - 1) / grain;
         }
-
-        template<class T> inline void bindTexture(const textureReference* tex, const PtrStepSz<T>& img)
```

Comment:
@cudawarped @asmorkalov Please take a look (cleanup after #22963)

Reply:
Looks good to me.